data/organisations.yml

```diff
@@ -31,3 +31,12 @@
   slug: rubyconf
   language: english
   youtube_channel_id: UCWnPjmqvljcafA0z2U1fwKQ
+- name: Friendly.rb
+  website: https://friendlyrb.com/
+  twitter: friendlyrb
+  slug: friendlyrb
+  youtube_channel_name: friendlyrb
+  youtube_channel_id: UC-PJCpt-6EgR4Ks6e0RSs1g # This is optional
+  kind: conference # Choose either 'conference' or 'meetup'
```

Comment:
```suggestion
  kind: podcast
```

Reply:
I actually wanted to upload the conference videos. Do I need to change this to podcast

Reply:
I actually wanted to upload the conference videos. Do I need to change this to podcast?

Reply:
ahhhh sorry my bad I read Friendly Show for what ever reason (tired or maybe because I listen to the latest episod recently) !!!! of course keep it as Conference.